packages/openapi/README.md

```diff
@@ -0,0 +1,72 @@
+  </a>
+  <a href="https://www.npmjs.com/package/@hyper-fetch/graphql">
+    <img src="https://custom-icon-badges.demolab.com/npm/v/@hyper-fetch/graphql.svg?logo=npm&color=e76f51" />
+  </a>
+  <a href="https://api.codeclimate.com/v1/badges/eade9435e75ecea0c004/test_coverage">
+    <img src="https://api.codeclimate.com/v1/badges/eade9435e75ecea0c004/test_coverage" />
+  </a>
+  <a href="https://github.com/BetterTyped/hyper-fetch">
+    <img src="https://custom-icon-badges.demolab.com/badge/typescript-%23007ACC.svg?logo=typescript&logoColor=white" />
+  </a>
+  <a href="https://github.com/BetterTyped/hyper-fetch">
+    <img src="https://custom-icon-badges.demolab.com/badge/-Axios-E10098?logo=axios&logoColor=white" />
+  </a>
+  <a href="https://www.npmjs.com/package/@hyper-fetch/axios">
+    <img src="https://custom-icon-badges.demolab.com/bundlephobia/minzip/@hyper-fetch/axios?color=64BC4B&logo=package" />
+  </a>
+</p>
+
+## About
+
+**`Hyper Fetch Axios`** is adapter for Hyper Fetch. It enables seamless communication with Axios, simplifying the
+retrieval and manipulation of data. Benefit from the convenience and efficiency of Hyper Fetch.
+
+## Features
+
+🔮 **Simple setup** - [Read more](https://hyperfetch.bettertyped.com/docs/guides/basic/setup)
+
+🎯 **Request cancellation** - [Read more](https://hyperfetch.bettertyped.com/docs/guides/advanced/cancellation)
+
+✨ **Window Focus/Blur Events** -
+[Read more](https://hyperfetch.bettertyped.com/docs/guides/react/core/window-focus-blur)
+
+🚀 **Queueing** - [Read more](https://hyperfetch.bettertyped.com/docs/guides/advanced/queueing)
+
+💎 **Automatic caching** - [Read more](https://hyperfetch.bettertyped.com/docs/documentation/core/cache)
+
+🪄 **Persistence** - [Read more](https://hyperfetch.bettertyped.com/docs/guides/advanced/persistence)
+
+🎊 **SSR Support** - [Read more](https://hyperfetch.bettertyped.com/docs/documentation/getting-started/environment)
+
+🔋 **Offline First** - [Read more](https://hyperfetch.bettertyped.com/docs/guides/advanced/offline)
+
+📡 **Built-in adapter** - [Read more](https://hyperfetch.bettertyped.com/docs/documentation/core/adapter)
+
+🧪 **Easy to test** - [Read more](https://hyperfetch.bettertyped.com/docs/documentation/getting-started/testing)
+
+🎟 **Authentication** - [Read more](https://hyperfetch.bettertyped.com/docs/guides/basic/authentication)
+
+💡 **Prefetching** - [Read more](https://hyperfetch.bettertyped.com/docs/guides/advanced/prefetching)
+
+## Help us keep working on this project ❤️
+
+- [Become a Sponsor on GitHub](https://github.com/sponsors/prc5)
+
+## Sources
+
+- #### [Installation](https://hyperfetch.bettertyped.com/docs/documentation/getting-started/installation)
+- #### [Docs](https://hyperfetch.bettertyped.com/)
+- #### [API](https://hyperfetch.bettertyped.com/api/)
+- #### [NPM](https://www.npmjs.com/package/@hyper-fetch/graphql)
```

Comment:
Can we add section about how to use it in the readme? With npx command, with file, with remote url etc?